pkg/settings/migration.go

```diff
@@ -98,7 +98,9 @@ func (r *Migration) Load() (err error) {
 		return liberr.Wrap(fmt.Errorf("failed to find environment variable %s", VirtV2vImage))
 	}
 	r.VirtV2vDontRequestKVM = getEnvBool(VirtV2vDontRequestKVM, false)
-	if r.CDIExportTokenTTL, err = getPositiveEnvLimit(CDIExportTokenTTL, 0); err != nil {
+
+	// Set timeout to 8 hours instead of the default 2
```

Comment:
i think it worth more details, why we needed the increase of time?(on the commit as well)